Mathlib/Combinatorics/SimpleGraph/Clique.lean

```diff
@@ -732,6 +756,14 @@ instance [DecidableEq α] [DecidableRel G.Adj] {n : ℕ} {s : Finset α} :
     Decidable (G.IsNIndepSet n s) :=
   decidable_of_iff' _ (G.isNIndepSet_iff n s)
 
+/-- The embedding of an `n`-independent set of an induced subgraph of the subgraph `G` is an
+`n`-independent set in `G` and vice versa. -/
+theorem induce_isNIndepSet_iff {F : Set α} {s : Finset { x // x ∈ F }} {n : ℕ} :
+    ((⊤ : Subgraph G).induce F).coe.IsNIndepSet n ↑s ↔
+    G.IsNIndepSet n (Finset.map ⟨Subtype.val, Subtype.val_injective⟩ s) := by
+  simp only [isNIndepSet_iff, coe_map, card_map, and_congr_left_iff]
+  exact fun _ ↦ induce_isIndepSet_iff G
```

Comment:
I would expect something like this to work
```suggestion
  simp [isNIndepSet_iff, induce_isIndepSet_iff]
```

Reply:
I would expect the same, but there is something really odd happening. your suggestion of 
`simp [isNIndepSet_iff, isIndepSet_induce]`
cannot prove the goal, but
` simp [isNIndepSet_iff, (isIndepSet_induce)]`
can! I did not believe my eyes, but the ci build seems to agree with me (see the latest two commits). I am now very confused.